sdk/spring-sdk/src/test/java/kalix/springsdk/testmodels/subscriptions/PubSubTestModels.java

```diff
@@ -93,6 +93,7 @@ public Action.Effect<Integer> methodTwo(Integer number) {
   }
 
   @Subscribe.EventSourcedEntity(value = CounterEventSourcedEntity.class, ignoreUnknown = true)
+  @Publish.Stream(id = "")
```

Comment:
```suggestion
```

Reply:
That was intentional. The goal was to validate that empty ID is not accepted, but that's probably something odd here. The class name is not exposing it. Either this is not being used or the validation was removed. 

I will check.